packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -339,6 +340,11 @@ export interface RepositoryProps {
    * @default Duration.days(15) for the database
    */
   readonly backupOptions?: RepositoryBackupOptions;
+
+  /**
+   * The security groups to add to the database.
+   */
+  readonly securityGroups?: ISecurityGroup[];
```

Comment:
Thinking a little out loud here ... There are three components in the Repository that need security groups.
1. EFS
2. DocDB
3. Repository installer instance/ASG.

It makes a certain amount of sense to expose the ability to provide a single SG for each of those; similar to the suggestion for the RenderQueue. Then, if we make sure that there are public object members for each of those things, the user can use the EFS/Cluster/ASG construct's own `addSecurityGroup()` and `IConnectable` interface to add/modify security group settings.